plugins/modules/azure_rm_notificationhub_info.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Praveen Ghuge (@praveenghuge), Karl Dasan (@karldas30)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_notificationhub_info
+short_description: Get Azure Notification Hub
+description:
+    - Get facts of Azure Notification Hub
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
```

Comment:
All parameter need define type
```suggestion
        required: True
        type: str
```